.github/workflows/elixir.yml

```diff
@@ -61,6 +61,29 @@ jobs:
       - name: Run format check
         run: mix format --check-formatted
 
+  audit:
+    name: Check dependencies for vulnerabilities
+    runs-on: u22-arm-runner
+    needs: [deps]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Elixir
+        id: beam
+        uses: erlef/setup-beam@v1
+        with:
+          otp-version: '27.2.1'
+          elixir-version: '1.18.2'
+      - name: Cache Mix
+        uses: actions/cache@v4
+        with:
+          path: deps
+          key: ${{ runner.os }}-mix-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
+      - name: Compile deps
+        run: mix deps.compile
+      - name: mix_audit check
+        run: mix deps.audit
```

Comment:
I think that it should be moved to separate action file and should probably be ran on schedule instead of being ran on PR.